pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -546,7 +552,10 @@ cdef class {{ message.full_name }}:
         {%- elif field.modifier == 'repeated' %}
         self._{{ field.name }}.extend(other_msg._{{ field.name }})
         {%- elif field.type == 'message' %}
-        if other_msg._{{ field.name }}._is_present_in_parent:
+        if other_msg._{{ field.name }} is not None and other_msg._{{ field.name }}._is_present_in_parent:
+            if self._{{ field.name }} is None:
+                self._{{ field.name }} = {{ field.message_name }}.__new__({{ field.message_name }})
+                self._{{ field.name }}._listener = <PyObject *>self
```

Comment:
I think we'll need to add

```python
self._{{ field.name }}.reset()
```

here, since otherwise the sub-message won't be initialised properly.